File: flickr/src/main/java/com/pablocampos/flickrsample/network/FlickrInterface.java
package com.pablocampos.flickrsample.network;

import com.pablocampos.flickrsample.model.ApiData;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface FlickrInterface {

	@GET("/services/feeds/photos_public.gne?format=json&nojsoncallback=?")
	Call<ApiData> getFlickrFeeds (@Query("tags") String tags);
}

File: flickr/src/main/kotlin/network/FlickrInterface.kt
package network

import model.ApiData
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface FlickrInterface {

	@GET("/services/feeds/photos_public.gne?format=json&nojsoncallback=?")
	fun getFlickrFeeds(@Query("tags") tags: String): Call<ApiData>
}
